contexts/myContext.js

```diff
@@ -0,0 +1,59 @@
+import { createContext, useState } from "react";
+import { useEffect } from "react";
+export const MyContext = createContext();
+
+export const MyProvider = ({ children }) => {
+  const [screenWidth, setScreenWidth] = useState(0);
+
+  const [currentbook, setCurrentbook] = useState("dieunendlichegeschichte");
+  const [theme, setTheme] = useState("quotes");
+  const [chooseCurrentBook, setChooseCurrentBook] = useState(false);
+  const [input1, setInput1] = useState("");
+  const [input2, setInput2] = useState("");
+  const [input3, setInput3] = useState("");
```

Comment:
I would change the names of the states here if possible like this it is not clear whats the purpose on the first sight. 😉